lib/chef/web/core/helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/web/core'
+require 'chef/web/core/component'
+require 'nokogiri'
+
+class Chef
+  module Web
+    module Core
+      module Helpers
+
+        def chef_logo(element=:div, opts={})
```

Comment:
I had to do this also, in the dummy Rails app. There might be something I can do in web-core to make it unnecessary in Rails; I'll check around.